dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.pages-4.0.feature

```diff
@@ -0,0 +1,10 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.jakarta.pages-4.0
+singleton=true
+-features=io.openliberty.servlet.api-6.1; apiJar=false, \
```

Comment:
I noticed servlet/faces/expressionLanguage/websocket API private features all contain the `io.openliberty.noShip-1.0` feature but this pages 4.0 API feature does not. Just checking to ensure that is correct.

Reply:
pages-4.0 private feature depends on servlet.api-6.1 in it so it already depends on a noship feature.  I purposely updated the other ones so that when eeCompatible-11.0 is moved to beta we do not need to update the rest of them at that time for the VisibilityTest that makes sure that noship features are not missed when moving something to beta.  I could add it here so that when servlet moves to beta and pages doesn't that we don't have to change it at that time.

Reply:
Thanks @jhanders34 if you can add it I wouldn't have an objection. Thanks.

Reply:
I also added it to the jakarta.cdi-4.0 one which would have hit it if expression language API feature went to beta before cdi did.